src/components/Login/Login.css

```diff
@@ -0,0 +1,73 @@
+/* MOBILE-FIRST VIEW */
+.Login {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  min-height: 100vh;
+  background-color: #3D157A;
+  /* background-image: linear-gradient(to bottom, #3D157A, #2D0F57, #3D157A); */
+}
+
+.greeting {
+  color: #BCBEF0;
+  font-family: 'Merienda', cursive;
+  font-size: xx-large;
+  margin-top: 10%;
+}
+
+.login-container {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  gap: 40px;
+  background-color: #131316;
+  box-sizing: border-box;
+  border: 1px solid #BCBEF0;
+  padding: 10%;
+}
+
+.login-button {
+  font-family: 'Arsenal', sans-serif;
+  font-size: large;
+  font-weight: 700;
+  cursor: pointer;
+  background-color: #3D157A;
+  color: #4AA3FC;
+  padding: 5px 15px 5px 15px;
+  width: 250px;
+}
+
+.login-button:hover {
+  background-color: #4AA3FC;
+  color: #3D157A;
+}
```

Comment:
Not 100% sure, but I think the `box-shadow` that gives the glow effect is missing?

Only thing I am a tiny bit torn on is with colors here of the back glow. I like the extra colors of the pink, but my mind mentally thinks as the button itself turns purple/text and blue button, the back glow would be a faint variation of these two colors instead. Not sure which I would prefer though, just a suggestion I would try.

Reply:
I'm a ding-dong. I didn't finish pushing it up. Hold, please...

Reply:
@Sykogst  Also, I haven't pushed it up yet, but here's how the buttons look with only the purple/blue. Do you like this better?
![Screenshot 2024-02-07 at 10 13 01 AM](https://github.com/Tavern-Keeper-2308/tavern-keeper-fe/assets/133910120/80ac588d-d5f9-4cf4-9332-d28de695ed80)

Reply:
Love this convo. Good catch Sam.